enterprise/server/remote_execution/containers/podman/BUILD

```diff
@@ -13,16 +13,25 @@ go_library(
         "//server/interfaces",
         "//server/util/alert",
         "//server/util/background",
+        "//server/util/disk",
         "//server/util/log",
         "//server/util/networking",
         "//server/util/random",
+        "//server/util/retry",
         "//server/util/status",
     ],
 )
 
 go_test(
     name = "podman_test",
     srcs = ["podman_test.go"],
+    args = [
+        # Note: these are used only for the Stats tests, which require running as root.
+        # These paths were tested on Ubuntu 20.04 + podman 3.4.2, but may be different for other setups.
```

Comment:
We are using podman 4 in dev/prod. Maybe worth testing on podman 4.

Reply:
Tested e2e with podman 4 in dev -- here is a chart using the metrics from #2139
 
![image](https://user-images.githubusercontent.com/2414826/174087901-7574f9a9-f98d-4040-b624-3e911be19f9b.png)